requirements.txt

```diff
@@ -21,14 +21,16 @@ py~=1.10.0
 pycryptodome>=3.9.8
 pyOpenSSL==19.1.0
 pytest-html==3.1.1
-pytest==6.2.5
+pytest==6.2.2 ; python_version <= "3.9"
+pytest==7.1.2 ; python_version >= "3.10"
 pyyaml==5.4
 requests==2.23.0
 scipy>=1.0; platform_system == "Linux" or platform_system == "Darwin" or platform_system=='Windows'
 seaborn>=0.11.1; platform_system == "Linux" or platform_system == "Darwin" or platform_system=='Windows'
 setuptools~=56.0.0
 testinfra==5.0.0
-jq>=1.1.2; platform_system == "Linux" or platform_system == "Darwin"
+jq==1.1.2 ; (platform_system == "Linux" or platform_system == "Darwin") and python_version <= "3.9"
+jq==1.2.2 ; python_version >= "3.10"
```

Comment:
Because we are using python 3.10 in our test environment and jq and pytest packages need another version for 3.10

Reply:
Requirements fixes for python3.10 environments were implemented in https://github.com/wazuh/wazuh-qa/pull/3107. If the error persists, please create a new issue.
Also, this test is an integration test.  This means that it will run in a python3.7 environment.
This does not fit in this PR.

Reply:
Ok